packages/react-menu/src/components/MenuGroup/useMenuGroup.ts

```diff
@@ -13,11 +13,11 @@ const mergeProps = makeMergePropsCompat<MenuGroupState>({ deepMerge: menuGroupSh
 /**
  * Given user props, returns state and render function for a MenuGroup.
  */
-export const useMenuGroup = (
+export function useMenuGroup(
```

Comment:
**q:** why do we need change func expresion to func declaration? AFAIR this doesn't impact declaration emit

Reply:
Sad, but it impacts and produces `import()` that breaks API extractor `import("../MenuDivider").MenuDividerProps`. Compare input and output below 😥

(_Another question, why it imports from **`../MenuDivider`**_)

----

#### Input

```tsx
export function useMenuGroup(
  props: MenuGroupProps,
  ref: React.Ref<HTMLElement>,
  defaultProps?: MenuGroupProps
): MenuGroupState {}
//
export const useMenuGroup = (
  props: MenuGroupProps,
  ref: React.Ref<HTMLElement>,
  defaultProps?: MenuGroupProps
): MenuGroupState => {};
```

#### Output

```tsx
export declare function useMenuGroup(
  props: MenuGroupProps,
  ref: React.Ref<HTMLElement>,
  defaultProps?: MenuGroupProps
): MenuGroupState;
//
export declare const useMenuGroup: (
  props: MenuGroupProps,
  ref: React.Ref<HTMLElement>,
  // 👇 adds `import()` and breaks API extractor
  defaultProps?: import("../MenuDivider").MenuDividerProps | undefined
) => MenuGroupState;
```

Reply:
_Technically, we can restore 63f75c8, but I like more to use `function` in this case than define a type for `const`._
Anyway, I am not personally attached to any solution, I am more disturbed that we are fixing code to pass tooling restrictions 😪

Reply:
hmm. I tested this locally to confirm, unfortunately I wasn't able to reproduce the difference being affected by ts declaration emit (expression vs declaration).

The troublesome emit is caused by following https://github.com/microsoft/fluentui/pull/17932/files#r620373707 (wait for it to load)